### PR TITLE
Move Minkelite dependency from mesh-models to PM

### PR DESCRIPTION
**Pull request by kraman**
Connect to strongloop-internal/scrum-nodeops#563

### CODE REVIEW

**Comment by kraman** [comment]
Depends on https://github.com/strongloop/strong-mesh-models/pull/58

File: lib/server.js [hunk 1]
```diff
@@ -64,13 +74,27 @@ function Server(options) {
     console: console,
     server: this,
   });
-  var meshOptions = {};
+  this.minkelite = null;
```
Comment (sam-github) [inline lib/server.js]: `_minkelite`, its private, right?

**Comment by sam-github** [comment]
minkelite should be private, other than that, LGTM

File: test/test-server.js [hunk 2]
```diff
@@ -200,7 +200,8 @@ tap.test('server test', function(t) {
         cmd: 'started',
         appName: 'test-app',
         agentVersion: '1.0.0',
-        pid: 1234
+        pid: 1234,
+        startTime: +(new Date())
```
Comment (rmg) [inline test/test-server.js]: More clear (and a few micro-ops faster) as `Date.now()`.

File: lib/server.js (hunk 1 above)
Comment (rmg) [inline lib/server.js]: Doesn't even need to live on the object, the only references are inside this constructor.

**Comment by rmg** [comment]
strong-mesh-models dep should be bumped to correspond with its major being bumped. See https://github.com/strongloop/strong-mesh-models/pull/58#issuecomment-104009233.

File: test/test-server.js (hunk 2 above)
Comment (rmg) [inline test/test-server.js]: Turns out I had already made this change in the docker branch and just merged that change as part of a cleanup PR, so you'll have to rebase and can probably get rid of this hunk in the process anyway.

**Comment by kraman** [comment]
@slnode test please

**Comment by kraman** [comment]
@slnode test please

**Comment by kraman** [comment]
@slnode test please

**Comment by kraman** [comment]
@slnode test please

**Comment by kraman** [comment]
Closing, added as separate comment on #213

**Comment by sam-github** [comment]
@kraman I don't think merging and abandoning PRs is helping us. The lack of this PR blocks docker testing by @cgole, and its not related to the tracer stuff.

Doing a rebase before merge is pretty easy, isn't it?

**Pull request closed without merging**